prosemirror-jdita/src/schema.ts

```diff
@@ -74,23 +102,45 @@ export const SCHEMA_CONTENT: Record<string, [content: string, groups: string]> =
   video: ['desc? media_source* media_track*', 'simple_blocks fig_blocks list_blocks all_blocks'],
   xref: ['common_inline*', 'all_inline'],
 }
+
+/**
+ * `SCHEMA_CHILDREN` is a map of special children for certain media nodes
+ */
 export const SCHEMA_CHILDREN: Record<string, (type: ChildTypes) => string[]> = {
   video: type => ['media-source', 'media-track', 'desc'],
   audio: type => ['media-source', 'media-track', 'desc'],
 }
+
+/**
+ * `IS_MARK` is containing an array of inline markup
+ * bold, italic, underline, subscript, superscript
+ */
 export const IS_MARK = ['b', 'i', 'u', 'sub', 'sup'];
 
+/**
+ * `SchemaNode` is a representation of a node in the schema
```

Comment:
`SchemaNode is a representation of a node` -> `A node`

Reply:
Fixed with 3f29158.